sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/telemetry/Telemetry.scala

```diff
@@ -58,17 +58,24 @@ object Telemetry {
   }
 }
 
-class Telemetry(serviceName: String, system: ActorSystem) {
+class Telemetry[T](serviceName: String, system: ActorSystem, commandType: T) {
```

Comment:
Maybe make it an internal ADT/enum for the different types of components instead of an arbitrary typed param? 

Also, it's not really the type of the command, it's rather something like "component category" (I guess "component type" is already used for the user categorising their concrete types of components).

Reply:
Of course. I didn't like it but didn't came up with anything better.

Reply:
done